### DISCUSSION TITLE
assigning mutations to a certain interval

### DISCUSSION THREAD

**Discussion by mshpak76** [MesserLab/SLiM#383] [General]
For reasons related to a previous problem that I had with overlapping conversion tracts for my model parameters, I am simulating model chromosomes of length 15,000 and only sampling the middle interval of 5000 (i.e. 4999-9999). Since my simulations introduce beneficial mutations, it is important that they are introduced in this interval of interest, however, I'm not certain of how to do this. Namely, if I have e.g.

    x = rdunif(1,0,14999);
    target = sample(p3.genomes, 1, replace=F);
    target.addNewDrawnMutation(m2, x);

Is there some way to force the mutation at a site in [4999,9999]?

**Comment by bhaller** [comment]
This ended up on slim-discuss (which is a better case for questions like this).  See https://groups.google.com/g/slim-discuss/c/2Uyz4fk8-2A/m/s_TsapkgAQAJ.  Closing.
